plugin.js

```diff
@@ -61,14 +61,17 @@ function plugin (fastify, options, next) {
     return cookie.parse(cookieHeader, options.parseOptions)
   })
   fastify.decorateRequest('cookies', null)
+  fastify.decorateRequest('unsignCookie', function unsignCookieRequestWrapper (value) {
```

Comment:
I didn't extract the call to `signer.unsign` in a generic function to leave as much information as possible for the call stack in case of an error.